src/pro/sky/java/course1/Main.java

```diff
@@ -1,8 +1,112 @@
 package pro.sky.java.course1;
 
+import java.util.Arrays;
+
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+        int[] arr = generateRandomArray();
+        for (int i = 0; i < arr.length; i++) {
+            System.out.println(arr[i]);
+        }
+        // work 1
+        int total = 0;
+        for(int i=0;i<arr.length;i++){
+            total = total + arr[i];
+        }
+        System.out.println("Сумма трат за месяц составила " + total + " рублей");
+
+        // work 2
+        int maxSum = arr[0];
+        int minSum = arr[0];
+        for(int i=1;i<arr.length;i++){
+            if(arr[i]>maxSum)
+                maxSum = arr[i];
+            else if(arr[i]<minSum)
+                    minSum = arr[i];
+        }
+
+        System.out.println("Минимальная сумма трат за день составила " + minSum + " рублей");
+        System.out.println("Максимальная сумма трат за день составила " + maxSum + " рублей");
+
+        // work 3
+        double argSum = total/arr.length;
```

Comment:
total и arr.length имеют тип int, поэтому само деление будет целочисленным, без дробной части. Чтобы её учитывать, нужно изменить делимое или делитель на тип double, например
```suggestion
        double argSum = total/(arr.length * 1.0);
```